metadata-ingestion/src/datahub/cli/put_cli.py

```diff
@@ -104,11 +109,12 @@ def platform(
         logoUrl=logo,
     )
     datahub_graph = get_default_graph()
-    datahub_graph.emit(
-        MetadataChangeProposalWrapper(
-            entityUrn=str(platform_urn), aspect=data_platform_info
-        )
+    mcp = MetadataChangeProposalWrapper(
+        entityUrn=str(platform_urn),
```

Comment:
is this change intended?

Reply:
Yes, it was the linter after I added an extra parameter.